### ISSUE TITLE
UnamBinder and Silent Crypto Miner

### ISSUE THREAD

**Issue by uscgbuzzman** [UnamSanctam/SilentCryptoMiner#1484]
Does UnamBinder works with all Silent Crypto Miner features? (Stealth, Process Killer, Idle mining, all the startup, run as administrator, add defender exclusions, disable windows defender, critical processes, disable sleep, run miner after install and install rootkit)

**Comment by UnamSanctam** [comment]
Yes, to ensure full compatibility with any bound programs what it (and most binders) do is to store the programs inside the code, and then when run it writes the file to the disk and executes it.
